defaults/main.yml

```diff
@@ -35,7 +35,7 @@ certbot_create_command: >-
   --email {{ cert_item.email | default(certbot_admin_email) }}
   {{ '--webroot-path ' if certbot_create_method == 'webroot'  else '' }}
   {{ cert_item.webroot | default(certbot_webroot) if certbot_create_method == 'webroot' else '' }}
-  -d {{ cert_item.domains | join(',') }}
+  --domains {{ cert_item.domains | join(',') }} --expand
```

Comment:
Do you believe the `--expand` option should be wrapped into another flag which determines its presence?

We are so used to using it all across the board, that we wouldn't even bother to omit it ;]. Do you see any negative impact on just having it as a default?